01-data-structures/02-stacks-and-queues/myqueue/myqueue.rb

```diff
@@ -5,14 +5,21 @@ class MyQueue
   def initialize
     @queue = Array.new
     @head = @queue[0]
+    @tail = @queue[0]
   end
 
   def enqueue(element)
+    @queue << element
+    @head = @queue[0]
+    @tail = element
   end
 
   def dequeue
+    @queue.delete_at(0)
+    @head = @queue[0]
   end
 
   def empty?
+    @queue === [] ? true : false
```

Comment:
You do not need the ternary here. `@queue === []` already results in `true` or `false`; there is no need to check it just to return `true` or `false` yourself.